build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
   `java-gradle-plugin`
   id("org.hypertrace.repository-plugin") version "0.4.0"
   id("org.hypertrace.ci-utils-plugin") version "0.3.0"
+  id("org.hypertrace.code-style-plugin") version "latest.release"
```

Comment:
We always want versions pinned. By latest version, I meant the actual version that was most recent :) Otherwise a breaking change could break the build.